### PR TITLE
adds a surgery tray to surgery supplies medical order, removes surgery duffelbags entirely

### PR DESCRIPTION
**Pull request by CoiledLamb**
## About The Pull Request
the surgery supplies order from the medical order console still included a surgical duffel, so i axed it, and replaced it with a surgery tray. 
also axes the surgical duffel from the code, as it's no longer necessary to have. as well as it's prefilled morgue variant
syndicate surgery duffelbags still work and are untouched. for flavor reasons
## Why It's Good For The Game
the original switch to surgery trays was done via update paths, so there really should not be any more of these duffels laying around in code, primed to confuse mappers. this brings a bit more consistency to medical orders, and removes that source of potential confusion.
## Changelog
:cl:

del: removes surgical duffelbags
fix: the surgery supply order now comes with a surgery tray
/:cl:

### CODE REVIEW

**Comment by EricZilla** [comment]
You may want to leave it in the code for downstream maps that haven't updated, but at the same time that's funny

**Review by tf-4 — COMMENTED** [review]
Remember to edit the description of the cargo crate to not claim it'll contain a duffel

**Comment by CoiledLamb** [comment]
> Remember to edit the description of the cargo crate to not claim it'll contain a duffel

will do, thanks

**Comment by CoiledLamb** [comment]
> You may want to leave it in the code for downstream maps that haven't updated, but at the same time that's funny

surgery trays mapedits are done with an update paths script so if they have the original PR it'll be done automatically. if they don't have it they should not merge this one.

File: code/modules/cargo/packs/medical.dm [hunk 1]
```diff
@@ -123,11 +123,11 @@
 /datum/supply_pack/medical/surgery
 	name = "Surgical Supplies Crate"
 	desc = "Do you want to perform surgery, but don't have one of those fancy \
-		shmancy degrees? Just get started with this crate containing a medical duffelbag, \
+		shmancy degrees? Just get started with this crate containing a DeForest surgery tray, \
```
Comment (JohnFulpWillard) [inline code/modules/cargo/packs/medical.dm:126]: who is DeForest

Comment (CoiledLamb) [reply code/modules/cargo/packs/medical.dm:126]: in universe medical company that I like referencing. you can see it on the surgery tray object itself, on shipping containers, and in the common core

Comment (CoiledLamb) [reply code/modules/cargo/packs/medical.dm:126]: here's a link to the page that talks abt em. https://github.com/tgstation/common_core/blob/master/Governments%20and%20Organisations/Corporations/Corporation%20Bible.md#deforest-medical-corporation
looking back they might've been better as interdyne, but I felt like interdyne gets all the lore. so I wanted to share the love a little

**Pull request merged**